hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/model/JpaTargetTag.java

```diff
@@ -29,52 +32,27 @@
 import org.eclipse.persistence.descriptors.DescriptorEvent;
 
 /**
- * A {@link TargetTag} is used to describe Target attributes and use them also
- * for filtering the target list.
- *
+ * A {@link TargetTag} is used to describe Target attributes and use them also for filtering the target list.
  */
+@NoArgsConstructor // Default constructor for JPA.
+@ToString(callSuper = true)
 @Entity
 @Table(name = "sp_target_tag", indexes = { @Index(name = "sp_idx_target_tag_prim", columnList = "tenant,id"),
         @Index(name = "sp_idx_target_tag_01", columnList = "tenant,name") }, uniqueConstraints = @UniqueConstraint(columnNames = {
                 "name", "tenant" }, name = "uk_targ_tag"))
 public class JpaTargetTag extends JpaTag implements TargetTag, EventAwareEntity {
-    private static final long serialVersionUID = 1L;
 
-    @CascadeOnDelete
-    @ManyToMany(mappedBy = "tags", targetEntity = JpaTarget.class, fetch = FetchType.LAZY)
-    private List<Target> assignedToTargets;
+    @Serial
+    private static final long serialVersionUID = 1L;
 
-    /**
-     * Constructor.
-     *
-     * @param name
-     *            of {@link TargetTag}
-     * @param description
-     *            of {@link TargetTag}
-     * @param colour
-     *            of {@link TargetTag}
-     */
     public JpaTargetTag(final String name, final String description, final String colour) {
         super(name, description, colour);
     }
 
-    public JpaTargetTag() {
-        // Default constructor for JPA.
-    }
-
-    public List<Target> getAssignedToTargets() {
```

Comment:
Not sure.. but i cannot find any previous usage of this even before removal now..

Reply:
maybe it wasn't used but since there was a field - I suppose it was just exposed for completenes